release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -87,6 +87,27 @@ development, and testing.
 For more information, see
 xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installation-three-node-cluster_installing-bare-metal[Running a three-node cluster].
 
+[id="ocp-4-5-disconnected-cluster-upgrade-improvements"]
+==== Disconnected cluster upgrade improvements
+
+During the upgrade process for a disconnected cluster, the Cluster Version
+Operator (CVO) now downloads the release image for a specific {product-title}
+version and verifies whether the image has been compromised. This provides a way
+to download the signature and provide it to the CVO for image verification in a
+restricted network environment.
```

Comment:
> the Cluster Version Operator (CVO) now downloads the release image for a specific {product-title}
version and verifies whether the image has been compromised

It was always the case. However previously CVO could only verfiy the images if it has access to `mirror.openshift.com` or to `https://storage.googleapis.com/openshift-release/official/signatures`. Hence customers had to use `--force` while upgrading disconnected environments. 

So as part of 4.5 we have added feature where CVO can verify the release payload if the image signature is available as a ConfigMap. So customers do not need to `--force` anymore.

Reply:
Suggestion for the above text

```
CVO can now verify the release images if the image signature is available as a ConfigMap in the cluster during the upgrade process for a disconnected cluster.  This removes the need for using `--force` flag while doing upgrades in disconnected environment. 
```